test/fixtures/templater/jinja_c_dbt/dbt_builtins_test.sql

```diff
@@ -0,0 +1,11 @@
+{% test my_cool_test(model, column_name, kwarg1=none, kwarg2=none) %}
+SELECT {{ column_name }} FROM {{ model }}
```

Comment:
The change broadly looks pretty sensible, but i'm not close to this part of the project. Why does this produce no SQL?

Reply:
@WittierDinosaur - these tests are basically like macros. You can see [the dbt docs](https://docs.getdbt.com/guides/best-practices/writing-custom-generic-tests) for details, but you define them like this and then refer to them in your various dbt schema yaml files to run the tests. So defining the test produces no sql itself.

Reply:
Ah gotcha, that makes sense